logging_utilities/formatters/json_formatter.py

```diff
@@ -114,6 +114,7 @@ def __init__(
         fmt=None,
         datefmt=None,
         style='%',
+        *,
```

Comment:
?

Reply:
Same as above, this says that only the first three parameters can be passed as positional parameters. This means that the other parameters could also be rearanged later without breaking anything. However now that you mention it I realize that this is a breaking change, so I can also revert that if you think its better.

Reply:
Ok that's something I didn't know about python :-) good to know, but is this also supported by older python version ?
Would be ok in new code but I would not change existing code.